mesop/server/config.py

```diff
@@ -11,9 +11,13 @@
 class Config(BaseModel):
   """Mesop app configuration."""
 
-  state_session_backend: Literal["memory", "file", "firestore", "none"] = "none"
+  state_session_backend: Literal[
+    "memory", "file", "firestore", "sql", "none"
+  ] = "none"
   state_session_backend_file_base_dir: Path = Path("/tmp")
   state_session_backend_firestore_collection: str = "mesop_state_sessions"
+  state_session_backend_sql_connection_uri: str = ""
+  state_session_backend_sql_table: str = "mesop_state_session"
```

Comment:
I would use the same str as you do for firestore "mesop_state_sessions" for consistency (I think it's fine to pluralize SQL table names b/c I think conventions vary widely for table naming)

Reply:
Heh, that's true. I was on the fence about this and ended up going for my preference non-plural table names. However, I agree that it would be better to be consistent with how we name the firestore collection.